azure-devops/azext_devops/dev/admin/banner.py

```diff
@@ -54,7 +54,7 @@ def banner_add(message, banner_type=None, id=None, expiration=None, organization
                        unique identifier is automatically created if one is not specified.
     :type id: str
     :param expiration: Date/time when the banner should no longer be presented to users. If not set, the banner does not
-                       automatically expire and must be removed with the remove command.
+                       automatically expire and must be removed with the remove command. Example : "2012-01-19 17:21:00 UTC"
```

Comment:
Here is the issue for filed for this - https://github.com/Microsoft/azure-devops-cli-extension/issues/225 

That seems like a long format. There will be easier/smaller date formats too that we support. Can we give another example with smaller date format.